schematic/schemas/data_model_relationships.py

```diff
@@ -0,0 +1,222 @@
+                    'edge_rel': False,
+                    'required_header': True,
+                    'node_attr_dict':{'default': get_attribute_display_name_from_label,
+                                    'standard': get_attribute_display_name_from_label,
+                                    },
+                },
+               'label':{
+                    'jsonld_key': 'rdfs:label',
+                    'csv_header': None,
+                    'node_label': 'label',
+                    'type': str,
+                    'edge_rel': False,
+                    'required_header': False,
+                    'node_attr_dict':{'default': get_label_from_display_name,
+                                    'standard': get_label_from_display_name,
+                                    },
+                },
+                'comment': {
+                    'jsonld_key': 'rdfs:comment',
+                    'csv_header': 'Description',
+                    'node_label': 'comment',
+                    'type': str,
+                    'edge_rel': False,
+                    'required_header': True,
+                    'node_attr_dict':{'default': 'TBD'},
+                },
+                'rangeIncludes': {
+                    'jsonld_key': 'schema:rangeIncludes',
+                    'csv_header': 'Valid Values',
+                    'edge_key': 'rangeValue',
+                    'jsonld_direction': 'out',
+                    'edge_dir': 'out',
+                    'type': list,
+                    'edge_rel': True,
+                    'required_header': True,
+                },
+                'requiresDependency': {
+                    'jsonld_key': 'sms:requiresDependency',
+                    'csv_header': 'DependsOn',
+                    'edge_key': 'requiresDependency',
+                    'jsonld_direction': 'out',
+                    'edge_dir': 'out',
+                    'type': list,
+                    'edge_rel': True,
+                    'required_header': True,
+                },
+                'requiresComponent': {
+                    'jsonld_key': 'sms:requiresComponent',
+                    'csv_header': 'DependsOn Component',
+                    'edge_key': 'requiresComponent',
+                    'jsonld_direction': 'out',
+                    'edge_dir': 'out',
+                    'type': list,
+                    'edge_rel': True,
+                    'required_header': True,
+                },
+                'required': {
+                    'jsonld_key': 'sms:required',
+                    'csv_header': 'Required',
+                    'node_label': 'required',
+                    'type': bool,
+                    'jsonld_default': 'sms:false',
+                    'edge_rel': False,
+                    'required_header': True,
+                    'node_attr_dict':{'default': False,
+                                      'standard': convert_bool_to_str,
```

Comment:
The way this is actually being used now is to convert str to bool or maybe more accurately extract a boolean value (bc will just pull bool if its recorded as a bool). Update the function name here, update schema utils and update data_model_nodes. Can pull the function directly from data_model_nodes.run_rel_functions"